algosdk/abi/method.py

```diff
@@ -0,0 +1,218 @@
+import json
+
+from Cryptodome.Hash import SHA512
+
+from algosdk import abi, constants, error
+
+
+TRANSACTION_ARGS = (
+    constants.PAYMENT_TXN,
```

Comment:
According to the latest ARC-4, [`txn` should also be acceptable](https://github.com/algorandfoundation/ARCs/blob/main/ARCs/arc-0004.md#transaction-types)